include/grpc/event_engine/event_engine.h

```diff
@@ -27,7 +27,6 @@
 #include <grpc/event_engine/memory_allocator.h>
 #include <grpc/event_engine/port.h>
 
-// TODO(hork): Define the Endpoint::Write metrics collection system
```

Comment:
This change seems unrelated to the rest of this PR.

Reply:
reverted